internal/controller/state/graph/grpcroute.go

```diff
@@ -444,3 +456,26 @@ func validateGRPCHeaderMatch(
 
 	return allErrs
 }
+
+func checkForUnsupportedGRPCFields(rule v1.GRPCRouteRule, rulePath *field.Path) field.ErrorList {
+	var ruleErrors field.ErrorList
+
+	if rule.Name != nil {
+		ruleErrors = append(ruleErrors, field.Forbidden(
+			rulePath.Child("name"),
+			"NGINX Gateway Fabric does not support \"SectionName\" field at the moment",
```

Comment:
We can simplify this error message - no need for the NGINX Gateway Fabric, and we should avoid language like "at the moment" because it implies we will support it soon, which may not be the case. Also SectionName is the name of the type which probably shouldn't be specified in the message - although to be fair, "name" is not very descriptive on it's own 😅 

Maybe:
```suggestion
			"rule names are not supported",
```